package.json

```diff
@@ -28,7 +28,7 @@
   },
   "devDependencies": {
     "@semantic-release/changelog": "^3.0.2",
-    "husky": "1.3.1"
+    "husky": "^2.3.0"
```

Comment:
Renovate will pin this again ([config](https://github.com/projectcaluma/renovate-config/blob/master/renovate.json#L36)).

[This just happened with the line above](https://github.com/adfinis-sygroup/semantic-release-config/pull/15/files).

Reply:
In that case we should maybe pin all of the dependencies... This was the only pinned dependency. Right?

Reply:
If you really want to use a semver range, you need to set this up in the renovate config.

Right now for dev deps it's:

```
      {
        "depTypeList": ["devDependencies"],
        "rangeStrategy": "pin"
      },
```

Reply:
I think it's best if we use pinned deps. Also I moved the changelog plugin to normal dependencies

Reply:
:+1: Did you forget to push?

Reply:
I amended the commit :wink:

Reply:
I see the force push, but it seems to have no changes....

Reply:
Turns out amending does not help if you forget to add the changes :joy: Should be pushed now

Reply:
:wink:

But it's still `"husky": "^2.3.0"`.

Reply:
We had a race condition. All good :smile: